RecoLocalMuon/CSCRecHitD/src/CSCMake2DRecHit.cc

```diff
@@ -140,37 +140,44 @@ CSCRecHit2D CSCMake2DRecHit::hitFromStripAndWire(const CSCDetId& id,
   float sigmaWithinTheStrip = -99.;
   int quality = -1;
   LocalPoint lp0(0., 0.);
+  int wglo = wg_left;
+  int wghi = wg_right;
+  float ymiddle;
+
+  // First wire of first wiregroup in cluster
+  wglo = layergeom_->middleWireOfGroup(wglo) + 0.5 - 0.5 * layergeom_->numberOfWiresPerGroup(wglo);
+  // Last wire in last wiregroup of cluster (OK if only 1 wiregroup too)
+  wghi = layergeom_->middleWireOfGroup(wghi) - 0.5 + 0.5 * layergeom_->numberOfWiresPerGroup(wghi);
 
   float stripWidth = -99.f;
   // If at the edge, then used 1 strip cluster only
   if (centerStrip == 1 || centerStrip == specs_->nStrips() || nStrip < 2) {
-    lp0 = layergeom_->stripWireIntersection(centerStrip, centerWire);
+    lp0 = (layergeom_->possibleRecHitPosition(float(centerStrip) - 0.5, wglo, wghi)).first;
+    ymiddle = lp0.y();
```

Comment:
this leads to a dead assignment
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-f461d2/10621/llvm-analysis/report-41ceff.html#EndPath
please check the code; if the `ymiddle` is not needed here, better remove it and perhaps even move its declaration from  L145 to the `else` below where it's actually used

Reply:
`ymiddle` was used earlier in `if` and `else`, but not anymore. I forgat to move the declaration. fixed now.